sass/themes/hb/_theme.scss

```diff
@@ -15,3 +15,7 @@ $desktop-title-color: $primary-color;
 $bus-color: red;
 $metro-color: blue;
 $rail-color: green;
+
+.leaflet-marker-icon {
```

Comment:
And this only effects the parking lots or other layers as well?

Reply:
All geojson layers.

Reply:
But I just wanted the parking lots to be blue, not all of them. The geojson layers have black icons, isn;t it?